assets/pages/new-contributions-landing/components/ContributionTextInput.jsx

```diff
@@ -90,8 +80,7 @@ ContributionTextInput.defaultProps = {
   autoFocus: false,
   max: undefined,
   min: undefined,
+  icon: null,
```

Comment:
I think this can be dropped entirely

Reply:
While I'm here should I remove the default props for all the other required props? I think we just need them for optional props